File: C-Program/add_num.c
#include <stdio.h>

/* this program takes in 2 command line arguments
 * which must be integers and prints the sum of them.
 */

int main( int argc, char *argv[] )  {
     int A = 0;
     int B = 0;
     sscanf(argv[1],"%d", &A);
     sscanf(argv[2],"%d", &B);
     printf("%d \n", A + B);
}

File: Java-Program/add_num.java
package simpleton;

/* this program returns takes in 2 arguments 
 * which must be integers and returns the sum of them. 
 */

public class add_num {

	public static void main(String[] args) {
		if (args.length == 2) {
			int a = Integer.parseInt(args[0]);
			int b = Integer.parseInt(args[1]);

			System.out.println(a + b); 
		}

	}

}